dev/com.ibm.websphere.appserver.features/build.gradle

```diff
@@ -191,151 +222,123 @@ task publishFeatureResources {
             classpath {
-                fileset(dir: "${rootDir}/com.ibm.ws.wlp.mavenFeatures/lib/") {
-                    include(name: 'LibertyFeaturesToMavenRepo-0.0.1-SNAPSHOT.jar')
+                fileset(dir: project(':wlp-mavenRepoTasks').buildDir) {
+                    include(name: 'wlp-mavenRepoTasks.jar')
                 }
-                fileset(dir: "${rootDir}/com.ibm.ws.logging.core/build/libs") {
-                    include(name: 'com.ibm.ws.logging.core.jar')
-                }
-                fileset(dir: "${rootDir}/com.ibm.ws.kernel.feature.core/build/libs") {
+                fileset(dir: project(':com.ibm.ws.kernel.feature.core').buildDir) {
                     include(name: 'com.ibm.ws.kernel.feature.core.jar')
                 }
-                fileset(dir: "${rootDir}/cnf/mavenlibs") {
-                    include(name: "org.apache.aries.util-*.jar")
-          			include(name: "jsoup-*.jar")
-          			include(name: "osgi.core*.jar")       			
-               	}               
+                fileset(dir:project.file('new')){
+                    include (name:'javax.json-1.1.2.jar')
+                }
+                fileset(dir:project.file('new')){
+                    include (name:'plexus-utils-*.jar')
+                }
+               
+                fileset(dir:cnf.file('mavenlibs')) {
+                    include(name: "ant-*.jar")
+                    include(name: "commons-io-*.jar")
+                    include(name: "commons-lang-*.jar")
+                    include(name: "javax.json-*.jar")
+                    include(name: "maven-model-*.jar")
+                }
+               
             }
         }
-                        
-        ant.featuresToMavenRepo(inputDirPath:"${rootDir}/cnf/build/temp/esas",
-                   outputDirPath:"${rootDir}/cnf/build/temp/mavenArtifact",
+
+        project.delete('build/temp/mavenArtifact')
+        project.mkdir('build/temp/mavenArtifact')
+        ant.featuresToMavenRepo(inputDirPath: project.file('build/libs/repo'),
+                   outputDirPath: project.file('build/temp/mavenArtifact'),
                    openLibertyJson:file('build/temp/SingleJson.json'))
-       
-       // ant.delete() {
-      	//	fileset(dir: 'build/temp/esas') {
-	 	//		include(name: '**/*.esa')
-		//	}
-	 //	}
-        
-        }
+    }
+}
+
+task zipOpenLibertyMavenRepo(type: Zip){
+	dependsOn parent.subprojects.assemble
+	//dependsOn 'cnf:generateMavenArtifact'
+		
+	//	basename 'openliberty-mavenArtifact'
+		from "${rootDir}/com.ibm.websphere.appserver.features/build/temp/"
+		include "mavenArtifact/**"
+		archiveName 'openliberty-mavenArtifact.zip'
+		
+    	destinationDir distsDir
+
 }
+   // zipOpenLibertyMavenRepo.dependsOn project("${rootDir}/cnf").tasks.generateMavenArtifact
+
+zipOpenLibertyMavenRepo.dependsOn generateMavenArtifact
+publish.dependsOn zipOpenLibertyMavenRepo
+
+publish.dependsOn generateMavenArtifact
```

Comment:
Is this line redundant, since the above lines already say publish dependsOn zipOpenLibertyMavenRepo which depends on generateMavenArtifact?